### PR TITLE
community/flatpak-builder: fix url, disable tests

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
@Cogitri

**Comment by Cogitri** [comment]
fwiw the tests failed for me with the same error as Travis

File: community/flatpak-builder/APKBUILD [hunk 1]
```diff
@@ -24,15 +26,9 @@ build() {
 		--libexecdir=/usr/lib/$pkgname \
 		--disable-static \
 		--with-dwarf-header=/usr/include/libdwarf
-
 	make
 }
 
-check() {
-	#make -k check
```
Comment (misery) [inline community/flatpak-builder/APKBUILD]: Why remove it? Should be fixed someday instead of "ignore it"

Comment (maxice8) [reply community/flatpak-builder/APKBUILD]: @fcolista (if i remember correctly) said it causes confusion to have check() defined with check disabled in options.

Comment (Cogitri) [reply community/flatpak-builder/APKBUILD]: Yup, @TBK also mentioned that

**Pull request closed without merging**